src/libPMacc/include/particles/tasks/TaskSendParticlesExchange.hpp

```diff
@@ -113,6 +113,15 @@ namespace PMacc
         virtual ~TaskSendParticlesExchange()
         {
             notify(this->myId, RECVFINISHED, NULL);
+            if(retryCounter != 0)
+            {
+                std::cerr << "Send buffer for particle species " <<
+                    ParBase::FrameType::getName() <<
+                    " is to small (max particles per send operation:" << maxSize <<
+                    ", direction:" << exchange <<
+                    ", retried:" << retryCounter <<
```

Comment:
Well my reasoning was, that if you get this, you'll get this a lot. So for 300k timesteps you may get this e.g. 3600k times. So saving 10 bytes in the string will save 36MB output and it is still enough to understand the error. But no strong opinion here as this already helps a lot